maker/src/routes.rs

```diff
@@ -140,3 +141,17 @@ pub async fn get_new_invoice() -> Result<(), HttpApiProblem> {
             .detail(format!("{e:#}"))
     })
 }
+
+#[rocket::get("/channel/list")]
+pub async fn get_channel_details() -> Result<(), HttpApiProblem> {
+    let list = get_channel_manager()
+        .map_err(|e| {
+            HttpApiProblem::new(StatusCode::INTERNAL_SERVER_ERROR)
+                .title("Failed to create lightning invoice")
+                .detail(format!("{e:#}"))
+        })?
+        .list_channels();
+
+    tracing::info!(?list, "Open channels: {}", list.len());
+    Ok(())
```

Comment:
❓ Why not returning the list instead of logging?

Reply:
good question! 
I chose this because `ChannelDetails` doesn't support `Serialize` out of the box, and I figured out that implementing a custom `Writer` or our own type would be an overkill right now, esp. when we're not really using it outside of testing.